pkg/cmd/admin/network/isolate_project.go

```diff
@@ -0,0 +1,77 @@
+package network
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	kerrors "k8s.io/kubernetes/pkg/util/errors"
+
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+)
+
+const (
+	IsolateProjectNetworkCommandName = "isolate-projects"
+
+	isolateProjectNetworkLong = `
+Isolate project network
+
+Allows project to isolate their network from other projects.`
```

Comment:
Needs to clarify that this only works with the multitenant plugin. ie, it only undoes "unisolate-projects".

(Inconsistency between "project" and "projects"; filename and most variables use the singular, but the actual option uses plural.)